Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -466,6 +466,19 @@ final class NotificationsCenterCell: UICollectionViewCell {
 
     // MARK: - Public
 
+    fileprivate func setupAccessibility(_ viewModel: NotificationsCenterCellViewModel) {
+        accessibilityAttributedLabel = viewModel.accessibilityText()
+        isAccessibilityElement = true
+        
+        let moreActionAccessibilityActionLabel = viewModel.isRead ? CommonStrings.notificationsCenterMarkAsUnread : CommonStrings.notificationsCenterMarkAsRead
+        let moreAction = UIAccessibilityCustomAction(name: "More", target: self, selector: #selector(tappedMoreAction))
```

Comment:
Is this what VoiceOver reads out? If so, instead of "More", should this be a translated string?

Reply:
Yes, great catch!